source/localizable/tutorial/autocomplete-component.md

```diff
@@ -102,6 +102,7 @@ Now, define your new controller like so:
 ```app/controllers/index.js
 export default Ember.Controller.extend({
   filteredList: null,
+  model: null,
```

Comment:
Can you amend the commit to remove this line? We generally don't declare the model property explicitly for routable controllers.